src/streaming/models/CmcdModel.js

```diff
@@ -348,13 +492,17 @@ function CmcdModel() {
 
 
     function _getGenericCmcdData() {
+        const cmcdParameters = getCmcdParametersFromManifest();
         const data = {};
 
         let cid = settings.get().streaming.cmcd.cid ? settings.get().streaming.cmcd.cid : internalData.cid;
 
-        data.v = CMCD_VERSION;
+        data.v = cmcdParameters.version ? cmcdParameters.version : CMCD_VERSION;
```

Comment:
In my opinion we can not simply take the version from the MPD here. We need to use the version that we use to generate the reporting fields.  In our case that is always version 1 atm. The `version` as specified in the MPD only tells use the _ highest CMCD version as defined in CTA-5004 that is accepted by the reporting server._

Reply:
@dsilhavy Thanks for your review and comments!! I did an update with the first set of changes and I'm gonna left questions in some of your comments for more details or to check if I understood well. There are changes that I didn't make yet because I'll need to clarify. 

In this case, is it okay to left `data.v = CMCD_VERSION` then?

Reply:
Thanks @matiasrb97 I am checking. Yes let's keep `data.v = CMCD_VERSION` here.